docs/general/editor.md

```diff
@@ -0,0 +1,23 @@
+# Setting Up Your Editor
+
+You can edit React Boilerplate using any editor or IDE, but there are a few extra steps that you can take to make sure your coding experience is as good as it can be.
+
+## VS Code 
```

Comment:
Feeling a bit sleazy submitting a docs page about editors that only explicitly mentions VS Code (since I'm also the owner of JS/TS support for VS Code). Let me know if there are any other editors that we should call out specifically on the new page